### PR TITLE
Add created and modified fields to user model

### LINKED ISSUE #95: Track user date_joined/created_at

### PR DESCRIPTION
**Pull request by rnegron**
Instead of adding one-off fields, I thought it would be nice to install the [django-model-utils](https://github.com/jazzband/django-model-utils) package and use some of its model fields now that the codebase supports it.

Closes #95.

### CODE REVIEW

**Comment by rnegron** [comment]
Just realized that this will add the `datetime` of when the migration is applied to all the user rows it updates by default. Probably not intended. Will fix.

**Comment by jpadilla** [comment]
> Just realized that this will add the datetime of when the migration is applied to all the user rows it updates by default. Probably not intended. Will fix.

This is not a huge deal. If we wanted to get closer to approx. we could use a user's first comment/review's created date.

**Comment by rnegron** [comment]
I went ahead and implemented your idea of using an existing user's first comment. I added the code into the migration file. Also removed `django_model_utils` since it was giving me trouble while running custom code in the migration

**Comment by jpadilla** [comment]
@rnegron nice! could you check why tests are failing?

**Comment by rnegron** [comment]
Oops, didn't mean to dismiss the review. Anyway, it should be good now!

File: notaso/users/fixtures/dummy.json [hunk 1]
```diff
@@ -2,12 +2,14 @@
   "model": "users.user",
   "pk": 1,
   "fields": {
-      "password": "pbkdf2_sha256$36000$nQmi1RHXkxtM$wLJEFsVaXDQ0uxEDsR30j72A4N8g0sHcXPLTvdlz4DM=",
+      "password": "password",
```
Comment (jpadilla) [inline notaso/users/fixtures/dummy.json]: @rnegron any reason for changing this?

**Comment by jpadilla** [comment]
@rnegron just deployed and ran migrations, all good. Thanks!

**Pull request merged**